multus/multus_test.go

```diff
@@ -1157,7 +1157,7 @@ var _ = Describe("multus operations", func() {
 		_, err = cmdAdd(args, fExec, nil)
 		Expect(fExec.addIndex).To(Equal(2))
 		Expect(fExec.delIndex).To(Equal(2))
-		Expect(err).To(MatchError("Multus: Err adding pod to network \"other1\": Multus: error in invoke Delegate add - \"other-plugin\": expected plugin failure"))
+		Expect(err).To(MatchError("Multus: error in adding pod to network \"other1\": delegateAdd: error in invoke Delegate add - \"other-plugin\": expected plugin failure"))
```

Comment:
This probably has to change now to pass the test, like, `Multus: error adding [...]`